src/instructlab/cli/model/chat.py

```diff
@@ -37,7 +37,6 @@
     "-c",
     "--context",
     cls=clickext.ConfigOption,
-    required=True,  # default from config
```

Comment:
why is this change needed?

Reply:
holdover from when ctx was being passed directly in the test
reverted